homeassistant/components/ezbeq/sensor.py

```diff
@@ -0,0 +1,69 @@
+"""Sensor platform for the ezbeq Profile Loader integration."""
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from . import EzBEQConfigEntry
+from .const import CURRENT_PROFILE, DEVICES
+from .coordinator import EzBEQCoordinator
+from .entity import EzBEQEntity
+
+
+@dataclass(frozen=True, kw_only=True)
+class EzBEQSensorEntityDescription(SensorEntityDescription):
+    """Describe EzBEQ sensor entity."""
+
+    value_fn: Callable[[EzBEQCoordinator], StateType]
+
+
+SENSORS: tuple[EzBEQSensorEntityDescription, ...] = (
+    EzBEQSensorEntityDescription(
+        key=CURRENT_PROFILE,
+        value_fn=lambda coordinator: coordinator.data.get(CURRENT_PROFILE)
+        if coordinator.data.get(CURRENT_PROFILE) != ""
+        else None,
+        translation_key=CURRENT_PROFILE,
+    ),
+    EzBEQSensorEntityDescription(
+        key=DEVICES,
+        # make a list of device names
+        value_fn=lambda coordinator: ", ".join(
+            f"{device.name}" for device in coordinator.devices
+        ),
+        translation_key=DEVICES,
+    ),
```

Comment:
This doesn't sound like a sensor. States have a max length and this sounds like it could reach it.

Either create entities for those devices and make them proper devices (via `via_device` you can link devices to other devices so you can see it as "connected via") or make this an action call (separate PR)